dogfood/coder/Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM rust:slim@sha256:9abf10cc84dfad6ace1b0aae3951dc5200f467c593394288c11db1e17b
 # Install rust helper programs
 # ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
 ENV CARGO_INSTALL_ROOT=/tmp/
-RUN cargo install exa bat ripgrep typos-cli watchexec-cli && \
+RUN cargo install typos-cli watchexec-cli && \
```

Comment:
review: `bat` was failing to build for me locally, but `bat`, `exa`, and `ripgrep` can now be installed via `apt`.